### ISSUE TITLE
Mapper fails to convert rowversion sql data type

### ISSUE THREAD

**Issue by fkucuk** [christiandelbianco/monitor-table-change-with-sqltabledependency#87]
My table contains a column with a sql data type "rowversion". ModelToTableMapper fails to convert it to ulong.

**Comment by christiandelbianco** [comment]
Hi, 
SQL Server _row version_ column type can be managed as shown in this text case:

https://github.com/christiandelbianco/monitor-table-change-with-sqltabledependency/blob/master/TableDependency.SqlClient.Test/RowVersionTypeTest.cs

Regards

**Comment by fkucuk** [comment]
Hi Christian, I am aware my first comment was lack of details.
We were trying to convert the sql's rowversion to ulong, but were getting irrelevant results.
Then we realized that our conversion code was causing this.

we got help from the following thread on StackOverflow.
[https://stackoverflow.com/questions/21146023/converting-sql-server-rowversion-to-long-or-ulong/38149945#38149945](https://stackoverflow.com/questions/21146023/converting-sql-server-rowversion-to-long-or-ulong/38149945#38149945)
